test/todos.js

```diff
@@ -197,4 +197,18 @@ describe(Todos, function() {
       actual.should.eql([2, 1]);
     });
   });
+
+  describe('#renumber', function() {
+    it('renumbers the todo items', function() {
+      var todos = new Todos(storage);
+
+      time.freeze();
```

Comment:
just curious, what does this line do and what's it for in this particular test setup?